l10n_it_account_tax_kind/model/account_tax_kind.py

```diff
@@ -7,17 +7,19 @@
 class AccountTaxKind(models.Model):
 
     _name = 'account.tax.kind'
-    _rec_name = 'display_name'
 
     code = fields.Char(string='Code', size=3, required=True)
     name = fields.Char(string='Name', required=True)
-    display_name = fields.Char(string='Name', compute='_compute_display_name')
 
-    @api.depends('code', 'name')
     @api.multi
-    def _compute_display_name(self):
```

Comment:
In order to have `display_name` recomputed when `code` or `name` change, you can add something like
```
@api.depends('code', 'name')
def _compute_display_name(self):
    """ Just add code and name dependencies for display_name recomputation"""
    return super(AccountTaxKind, self)._compute_display_name()
```

Reply:
Why would you do this? ```display_name``` is not stored nor viewable in any form

Reply:
yes but I see [here](https://github.com/odoo/odoo/blob/11.0/odoo/models.py#L1487) that it depends on the field defined in `_rec_name`, then why not adding the fields we need?

Reply:
@jackjack82 we can skip the dependencies as long as the test works